### DISCUSSION TITLE
请教下各位大佬，这个修改解决什么问题，是否必须同步到项目中？

### DISCUSSION THREAD

**Discussion by jtby** [cloudwu/skynet#1874] [Q&A]
偶然看到这个修改：[bugfix: skynet.coroutine](https://github.com/cloudwu/skynet/commit/ea129cd9f6cf9a220b27d565d49f0645b2a177e0)
不知道涉及到哪些问题，请教下各位大佬。
因为项目比较老，还是使用的v1.0.0版本的skynet，屎山项目直接升级到最新不太现实，是否有必要手动同步这个修改？

**Comment by hanxi** [comment]
profile 应该不影响游戏逻辑。
